src/base-components/accordion-base/panel-base.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+
+const PanelBase = React.forwardRef(({ children, classes, id, header, ExpandIcon, CompressIcon, alignIcon, open, setOpen, onChange }, ref) => {
+  const detailsNoHeight = classes.details.split(/\bh-\w+|\bp-\w+|\bpy-\w+/).map((r) => r.trim()).filter((r) => r).join(' ')
+  const Icon = open.includes(id) ? ExpandIcon : CompressIcon ? CompressIcon : ExpandIcon
+  const renderIcon = () => {
+    if (Icon) {
+      return (
+        <span className={`${classes.iconRoot}`}>
+          <Icon className={clsx(`${classes.icon} transition-transform duration-300 ease-in-out origin-center transform`, {
+            '-rotate-180': open.includes(id) && alignIcon === 'start',
+            'rotate-180': open.includes(id) && alignIcon === 'end',
+            'rotate-0': !open.includes(id),
+          })} />
+        </span>
+      )
+    }
+    return null
+  }
+
+  const handleClick = () => setOpen((prev) => {
+    if (prev.includes(id)) {
+      prev.splice(prev.indexOf(id), 1)
+      onChange(id, prev)
+      return [...prev]
+    }
+    onChange(id, [...prev, id])
+    return [...prev, id]
+  })
+
+  return (
+    <div ref={ref}>
+      <div
+        className={clsx(`${classes.header} cursor-pointer flex flex-row`, {
+          'justify-between': alignIcon === 'end',
+        })}
+        onClick={handleClick}
+      >
+        {alignIcon === 'start' && renderIcon()}
+        <span className='inline-block align-baseline'>{header}</span>
+        {alignIcon === 'end' && renderIcon()}
+      </div>
+      <div className={clsx('transition-height ease-in-out duration-300 overflow-y-hidden', {
+        [classes.details]: open.includes(id),
+        [`${detailsNoHeight} h-0`]: !open.includes(id),
+      })}>
+        {children}
+      </div>
```

Comment:
echoing from [chromatic comment](https://www.chromatic.com/pullrequest?appId=60d0cd6c298707003972699d&number=16&activeElementId=comment-thread-60ee00ed0f753b00499399d3):

> for panel content, should we have built-in padding to distinguish it visually from the header?

Reply:
@sallkall @YizhiCatherineZhang ^

Reply:
Good call, my bad I didn't match the top/bottom padding for the header. I will update the design file meanwhile @hyx131  could you add 10px all around (`padding: 10px;`). Should look roughly the same (currently it is `padding: 10px 10px 5px 10px` )

Reply:
Actually I did do 10px all around: 
![image](https://user-images.githubusercontent.com/33563070/125649854-62b67783-aaf4-44ce-b11d-d65c7197968e.png)
![image](https://user-images.githubusercontent.com/33563070/125649891-73a26f08-921e-4e4d-a939-a0f18352f132.png)

Reply:
I think colours are enough to distinguish, but maybe as an option we can have a `indented` attribute to achieve what you had in mind? Otherwise i don't think extra padding for the content section is always needed

Reply:
ok. in that case, let's defer this to future iterations to add `indented` flag as a new feature

Reply:
sounds good~